openedx/core/djangoapps/theming/helpers.py

```diff
@@ -196,17 +196,19 @@ def get_current_site_theme_dir():
     Returns:
          (str): theme directory for current site
     """
-    site = get_current_site()
-    if not site:
+    from edxmako.middleware import REQUEST_CONTEXT
+    request = getattr(REQUEST_CONTEXT, 'request', None)
```

Comment:
Is this the best way to access the request context at this location?  I've seen other approaches throughout the code as well -- for example via threading -- for an example you can see django_sites_extensions.middleware.CurrentSiteWithDefaultMiddleware and how it is used in Otto (the middleware is also in LMS)